storio-basic-sample-app/src/main/java/com/pushtorefresh/storio/basic_sample/Tweet.java

```diff
@@ -5,31 +5,34 @@
 
 import com.pushtorefresh.storio.sqlite.annotations.StorIOSQLiteColumn;
 import com.pushtorefresh.storio.sqlite.annotations.StorIOSQLiteType;
-
-
 // Just a class for demonstration, real Tweet structure is more complex.
 
 // This annotation will trigger annotation processor
 // Which will generate type mapping code in compile time,
 // You just need to link it in your code.
-@StorIOSQLiteType(table = TweetsTable.TABLE)
+@StorIOSQLiteType(table = "tweets")
```

Comment:
Nice! But I not sure that everyone will need generated class, maybe add flag to this annotation? Something like `@StorIOSQLiteType(table = "tweets", generateTableClass = false)` which will be true by default?